ios/Plugin/Plugin.swift

```diff
@@ -275,10 +275,10 @@ public class AdMob: CAPPlugin, GADBannerViewDelegate, GADInterstitialDelegate, G
      */
     @objc func prepareInterstitial(_ call: CAPPluginCall) {
         DispatchQueue.main.async {
-            var adUnitID = call.getString("adId") ?? "ca-app-pub-3940256099942544/4411468910"
+            var adUnitID = call.getString("adId") ?? "ca-app-pub-3940256099942544/1033173712"
             let isTest = call.getBool("isTesting") ?? false
             if isTest {
-                adUnitID = "ca-app-pub-3940256099942544/6300978111"
+                adUnitID = "ca-app-pub-3940256099942544/1033173712"
             }
```

Comment:
Maybe it would be better in this way?

```
let testingID = "ca-app-pub-3940256099942544/1033173712"; 
var adUnitID = testingID
let isTest = call.getBool("isTesting") ?? false

if !isTest {
    adUnitID = call.getString("adId") ?? testingID
}
```

So there is just one magic string for the testingID and if we request a testing ad no `getBool` call is done.

Reply:
In my personal opinion I like more the current approach keeping the if condition handling the development case. It will still get through `getBool` and `getString` for 99% of cases (I'm thinking of production users). What I see a nice to have from your snippet is having a let for testingID, this way it will prevent other mistakes. What do you think about this:

`
let testingID = "ca-app-pub-3940256099942544/1033173712" 
let isTest = call.getBool("isTesting") ?? false
var adUnitID = call.getString("adId") ?? testingID

if isTest {
    adUnitID = testingID
}
`

I can change it for all 3 scenarios to share the same approach (banner, reward, interstitial).

Reply:
Yes that is good compromise 👍